models/staging/education/staging_student_satisfaction_survey.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(
+  materialized='table'
+) }}
+
+WITH source_data AS (
+    SELECT
+        id,
+        data::jsonb AS json_data  -- Cast the data column to JSONB
```

Comment:
Is this needed ? I am guessing this is from one of our custom connectors. 
As far as i remember, the data column should be a jsonb.